source/transactions/transactions.rst

```diff
@@ -1416,8 +1407,10 @@ durable, which achieves the primary objective of avoiding duplicate commits.
 **Changelog**
```

Comment:
I have no idea why this spec adds bold formatting to section titles, but it seemed like too much to change and removing it for this one section would make it inconsistent with itself.